test/e2e/network/service.go

```diff
@@ -1179,6 +1179,37 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(err)
 	})
 
+	/*
```

Comment:
What is this? C? :slightly_smiling_face:

Reply:
I cherry picked the previous test, learning something new every day :)

https://golang.org/doc/effective_go.html#commentary